dev-packages/rollup-utils/plugins/bundlePlugins.mjs

```diff
@@ -140,6 +140,8 @@ export function makeTerserPlugin() {
           '_isShim',
           // This is used in metadata integration
           '_sentryModuleMetadata',
+          // _sentry_span_name_set_by_user is set by the user and is used to store the original span name
+          '_sentry_span_name_set_by_user',
```

Comment:
l: Since we do not mangle this, can we maybe shorten this a bit? Or actually, do we even need to `_` this, can we just use e.g. `sentry.spanName`, then we do not need to add this here at all - there is prior context for this as well, we use e.g. `SEMANTIC_ATTRIBUTE_SENTRY_PARENT_IS_REMOTE` which is `sentry.parentIsRemote`, which we also strip before we send stuff.

Reply:
Ah ok I didn't realize there was precedence for it so I wanted to treat it explicitly as a temporary attribute. In this case, I'll also just defined a semantic attribute const and use that without a leading `_`. Thanks!

Reply:
[64437ed](https://github.com/getsentry/sentry-javascript/pull/14291/commits/64437edc9877df21d24ca5287b6b0475e4300b0b)